verification/curator-service/api/src/schemas/sources.indexes.json

```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "name": "uploadsSortOrderIdx",
+        "key": {
+            "uploads.created": -1,
+            "name": -1
+        },
+        "partialFilterExpression": {
+            "uploads.0": {
+                "$exists": true
+            }
+        },
+        "collation": {
```

Comment:
I included it for the "name" field in the key. The docs say it's used for string comparisons, so I assume it would be useful in sorting: https://docs.mongodb.com/manual/tutorial/sort-results-with-indexes/#index-use-and-collation

Reply:
Collation is used for case sensitiveness which we use for searching the line list but not for sources, you can live with the default collation which is simpler imo.

Reply:
Also if you don't specify the collation in search this index won't be used, so you must remove this currently or change the code to use it.

Reply:
Okay, removed.